proton-c/src/tests/fuzz/fuzz-sniff-header.c

```diff
@@ -0,0 +1,15 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "core/autodetect.h"
+
+#ifdef __cplusplus
```

Comment:
The `LLVMFuzzerTestOneInput` function is to be visible externally. libFuzzingEngine provides a `main` function which calls it.